### DISCUSSION TITLE
how can i find resource folder

### DISCUSSION THREAD

**Discussion by am1752** [rustdesk/rustdesk#8264] [Q&A]
when i start debug at build.py, i see set res_dir is 'reources'.
but i can't find this folder.
how can i generate or find this folder?

**Comment by yez730** [comment]
i came cross the same issue #8263, found the clue here: https://stackoverflow.com/a/78504630

solved by adding a `rust-toolchain.toml` file with:
```
[toolchain]
channel = "1.75"
```
